### ISSUE TITLE
Promises are not resolved as described in docs

### ISSUE THREAD

**Issue by charlike** [PolymerLabs/lit-html#27]
As i understand from the docs this should be possible

```js
import { render, html } from "lit-html";

const Main = () => {
  const postTitle = window
    .fetch("http://jsonplaceholder.typicode.com/posts/1")
    .then(post => post.title);

  return html`<h1>Post: ${postTitle}</h1>`;
};

render(Main(), document.body);

```

https://codesandbox.io/s/Z8Xjx1l5

**Comment by justinfagnani** [comment]
Yes, that should work. It's only on master so far, not npm.

**Comment by justinfagnani** [comment]
Just released 0.5.0 with Promise support. Please reopen if that doesn't fix it.
